docs/reference/components/sinks/elasticsearch.cue

```diff
@@ -146,6 +146,15 @@ components: sinks: elasticsearch: {
 				default: "_doc"
 			}
 		}
+		endpoint: {
+			description: "The Elasticsearch endpoint to send logs to. This should be the full URL as shown in the example."
+			required:    true
+			warnings: []
+			type: string: {
+				default: null
```

Comment:
I think this `default` key cannot be there if `required` is `true`. I believe that is leading to the current CI failure.